### ISSUE TITLE
Codemod generates `blitz-server.ts` with errors

### ISSUE THREAD

**Issue by Nfinished** [blitz-js/blitz#3937]
### What is the problem?

The beta 14 codemod generates a bad `blitz-server.ts`, notably it omits the required `logger` property from `setupBlitzServer`'s options arg

### Paste all your error logs here:

```
Argument of type '{ plugins: BlitzServerPlugin<any, any, {}>[]; }' is not assignable to parameter of type 'SetupBlitzOptions'.
  Property 'logger' is missing in type '{ plugins: BlitzServerPlugin<any, any, {}>[]; }' but required in type 'SetupBlitzOptions'
```

### Paste all relevant code snippets here:

```ts
// blitz-server.ts (generated)

import { BlitzServerMiddleware } from 'blitz'
import { setupBlitzServer } from '@blitzjs/next'
import { AuthServerPlugin, PrismaStorage } from '@blitzjs/auth'
import { simpleRolesIsAuthorized } from '@blitzjs/auth'
import db from 'db'
import { authConfig } from './blitz-client'

export const { gSSP, gSP, api } = setupBlitzServer({
  plugins: [
    AuthServerPlugin({
      ...authConfig,
      storage: PrismaStorage(db),
      isAuthorized: simpleRolesIsAuthorized,
    }),
  ],
})

```

### What are detailed steps to reproduce this?

1. Run the codemod

### Run `blitz -v` and paste the output here:

```
Blitz version: 2.0.0-beta.14 (local)
macOS Monterey | darwin-x64 | Node: v16.14.0


 Package manager: npm

  System:
    OS: macOS 12.6
    CPU: (8) x64 Intel(R) Core(TM) i7-8569U CPU @ 2.80GHz
    Memory: 27.74 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.0 - /var/folders/34/c8tk0fm52hz6z0288x5npsp80000gp/T/yarn--1667262429779-0.6141956480544359/node
    Yarn: 1.22.19 - /var/folders/34/c8tk0fm52hz6z0288x5npsp80000gp/T/yarn--1667262429779-0.6141956480544359/yarn
    npm: 8.3.1 - ~/.asdf/plugins/nodejs/shims/npm
  npmPackages:
    @blitzjs/auth: 2.0.0-beta.14 => 2.0.0-beta.14
    @blitzjs/next: 2.0.0-beta.14 => 2.0.0-beta.14
    @blitzjs/rpc: 2.0.0-beta.14 => 2.0.0-beta.14
    @prisma/client: ^3.14.0 => 3.14.0
    blitz: 2.0.0-beta.14 => 2.0.0-beta.14
    next: 12.2.0 => 12.2.0
    prisma: ^3.14.0 => 3.14.0
    react: ^18.1.0 => 18.1.0
    react-dom: ^18.1.0 => 18.1.0
    typescript: ~4.7.2 => 4.7.2
```

**Comment by dillonraphael** [comment]
Ah good point. We released the logger property after the codemod. Will get this fixed.

**Comment by flybayer** [comment]
Fixed in #3939
